invenio_app_ils/ui/src/invenio_app_ils/common/components/ResultsSort/ResultsSort.js

```diff
@@ -1,29 +1,31 @@
-import React from 'react';
 import { ResultsPerPage, SortBy, SortOrder } from 'react-searchkit';
+import React, { Component } from 'react';
 
-export function ResultsSort(props) {
-  return props.searchConfig.SORT_BY.length ? (
-    <div>
-      <span>
-        Show&nbsp;
+export class ResultsSort extends Component {
```

Comment:
We are sort of going in circles with this one, we have to decide how our search actions are going to look like.

Reply:
I only changed it to be a component not a function. For the frontsite search I've create a new `SearchControls` component, because I've rearranged the filters etc, but I didn't remove the `ResultsSort because it is used in the backoffice.